lib/spack/spack/cmd/install.py

```diff
@@ -73,6 +77,11 @@ def setup_parser(subparser):
         '--fake', action='store_true', dest='fake',
         help="Fake install. Just remove prefix and create a fake file.")
 
+    subparser.add_argument(
+        '--install-status', '-I', action='store_true', dest='install_status',
+        help="Show spec before installing.")
```

Comment:
Because it takes 68 seconds instead of 34; and because I really want to use it routinely on every `spack install` or `spack setup` command.  That way, I can see what Spack is doing while it's doing it, and I don't have to type twice as much and wait twice as long before I can check whether things are going OK.  Why all the pushback on this one?  It doesn't affect anybody who doesn't want to use it.